neptune/exceptions.py

```diff
@@ -62,6 +62,13 @@ class NeptuneException(Exception):
     pass
 
 
+class AlphaProjectException(NeptuneException):
+    def __init__(self, project):
+        super(AlphaProjectException).__init__(
+            '{} is Alpha Neptune Project. Use neptune.alpha module from alpha version of neptune-client library'
```

Comment:
`is an Alpha Neptune Project` -- add **an**.

Also, I don't really know what to do with this exception. What is `alpha version of neptune-client library`? How to get it?